src/components/common/PageLayout/index.tsx

```diff
@@ -27,11 +26,17 @@ const isNoSidebarRoute = (pathname: string): boolean => {
 }
 
 const PageLayout = ({ pathname, children }: { pathname: string; children: ReactElement }): ReactElement => {
-  const noSidebar = isNoSidebarRoute(pathname)
+  const router = useRouter()
+  const [noSidebar, setNoSidebar] = useState<boolean>(isNoSidebarRoute(pathname))
   const [isSidebarOpen, setSidebarOpen] = useState<boolean>(true)
   let isAnimated = useDebounce(!noSidebar, 300)
   if (noSidebar) isAnimated = false
 
+  useEffect(() => {
+    const noSafeAddress = router.isReady && !router.query.safe
+    setNoSidebar(isNoSidebarRoute(pathname) || noSafeAddress)
```

Comment:
Afaik all the paths in `isNoSidebarRoute` are non-safe routes. Could we instead add the new settings path there as well or remove that check entirely and always depend on `!router.query.safe` to hide the sidebar?

Reply:
The settings path is the same – `/settigns`. Could be a new path tho, e.g. `/preferences`.

Reply:
Ah makes sense then if its the same page.